support-workers/src/test/scala/com/gu/support/workers/integration/CreateZuoraSubscriptionSpec.scala

```diff
@@ -65,7 +65,7 @@ class CreateZuoraSubscriptionSpec extends AsyncLambdaSpec with MockServicesCreat
   }
 
   it should "create a 6 for 6 Guardian Weekly subscription" in {
-    createSubscription(createGuardianWeeklySubscriptionJson(Quarterly, Some(DefaultPromotions.GuardianWeekly.NonGift.sixForSix)))
+    createSubscription(createGuardianWeeklySubscriptionJson(SixWeekly, Some(DefaultPromotions.GuardianWeekly.NonGift.sixForSix)))
```

Comment:
this was probably not working as intended for a while, but we didn't check the promo applied ok before.

Reply:
A nice proof that this is working!